account-kit/react/src/components/auth/sections/OAuth.tsx

```diff
@@ -8,7 +9,7 @@ type Props = Extract<AuthType, { type: "social" }>;
 
 // Not used externally
 // eslint-disable-next-line jsdoc/require-jsdoc
-export const OAuth = ({ ...config }: Props) => {
+export const OAuth = memo(({ ...config }: Props) => {
```

Comment:
this component is also super expensive to render and I haven't figure out why. This also saves a number of renders too